Dockerfile

```diff
@@ -0,0 +1,31 @@
+# extend node alpine base image
+FROM node:8.4.0-alpine
+
+ENV RENOVATE_VERSION 9.45.10
+
+MAINTAINER Rhys Arkins <rhys@arkins.net>
```

Comment:
Docker documentation says `MAINTAINER` is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). `LABEL` should be used instead.